oomph-vscode/syntaxes/oomph.tmLanguage.json

```diff
@@ -0,0 +1,65 @@
+{
```

Comment:
Please delete `oomph-vscode/syntaxes/.oomph.tmLanguage.json.swp`. It seems to be a nano swapfile